client/ayon_core/hosts/maya/api/alembic.py

```diff
@@ -0,0 +1,306 @@
+import json
+import logging
+import os
+
+from maya import cmds  # noqa
+
+from ayon_core.hosts.maya.api.lib import evaluation
+
+log = logging.getLogger(__name__)
+
+# The maya alembic export types
+ALEMBIC_ARGS = {
+    "attr": (list, tuple),
+    "attrPrefix": (list, tuple),
+    "autoSubd": bool,
+    "dataFormat": str,
+    "dontSkipUnwrittenFrames": bool,
```

Comment:
Should we remove this? `extract_alembic` takes no `kwargs` and also does not take this argument?
```suggestion
```